Types incompatible with TS 4.4

tested with TS 4.4.3 only, so far. I get compilation errors within redoodle types:

```
[error] TypeScript error node_modules/.pnpm/redoodle@2.6.1/node_modules/redoodle/lib/Store.d.ts:24:5 - error TS1169: A computed property name in an interface must refer to an expression whose type is a literal type or a 'unique symbol' type.

24     [Symbol.observable](): _ObservableLike<S>;
```

--- a/changelog/@unreleased/pr-79.v2.yml
+++ b/changelog/@unreleased/pr-79.v2.yml
@@ -1,0 +1,7 @@
+type: fix
+fix:
+  description: Fix symbol-observable typing issue that caused Typescript compilation
+    to fail for consumers that don't depend on `redux`
+  links:
+  - https://github.com/palantir/redoodle/pull/79
+  - https://github.com/palantir/redoodle/issues/63

--- a/src/index.ts
+++ b/src/index.ts
@@ -39,3 +39,4 @@ export { omit } from "./omit";
 export { reduceCompoundActions } from "./reduceCompoundActions";
 export { setWith } from "./setWith";
 export type { PayloadOf } from "./PayloadOf";
+export * from "./symbol-observable";

--- a/src/symbol-observable.ts
+++ b/src/symbol-observable.ts
@@ -1,0 +1,14 @@
+/**
+ * We get this type filled in globally from the symbol-observable package, but if consumers of
+ * redoodle don't consume either symbol-observable or redux (which also fills in this global type),
+ * then consumers of redoodle will fail Typescript compilation without skipLibCheck turned on.
+ * Filling the type in here so that it is emitted in redoodle's type declaration files.
+ */
+
+declare global {
+  export interface SymbolConstructor {
+    readonly observable: symbol;
+  }
+}
+
+export {};

